en-US/U.xml

```diff
@@ -182,10 +182,10 @@
 
 		</varlistentry>
 		 <varlistentry id="url">
-			<term>URL</term>
+			<term>URL, URI</term>
 			 <listitem>
 				<para>
-					<emphasis>n.</emphasis> Include the appropriate protocol, such as http, ftp, or https, at the beginning of URLs. That is, use http://www.redhat.com and not www.redhat.com.
+					<emphasis>n.</emphasis> Include the appropriate protocol, such as http, ftp, or https, at the beginning of URLs or URIs. That is, use http://www.redhat.com and not www.redhat.com.
```

Comment:
A minor nit; perhaps we can leave for another time:

In [URI (and URL) parlance](https://www.rfc-editor.org/rfc/rfc3986#section-3), that first component before the `:` is formally referred to as a _scheme_. 

Schemes are semantically different from network protocols, yet they tend to have the same name! So the `http` or `https` scheme in a URL/URI _is also the name of a protocol_. Contrast this with the`file` scheme, for example, which has no associated protocol. Also note that `:` and `//` are not part of the scheme either. Even those have names!

Still a minor nit for now. I can save for a separate issue if desired.